static/js/components/tiles/histogram_tile.tsx

```diff
@@ -170,10 +174,20 @@ function binData(
     // Get start time in the temporal granularity desired
     const eventDate = event.startDate.slice(0, format.length);
 
-    // Increment count in corresponding bin if event has at least
+    // Get value of property to aggregate
+    let eventValue = 0;
+    if (property) {
+      // default to 0 if property can't be found in display props
+      eventValue = event.displayProps[property] ?? 0;
```

Comment:
maybe use `||` to catch other false cases

Reply:
Changed to ||.